project/Vagrantfile

```diff
@@ -31,16 +31,20 @@ end
 @properties = load_properties("project.properties")
 @ml_version = ENV['ML_VERSION'] || @properties['ML_VERSION'] || "7"
 @nr_hosts = (ENV['NR_HOSTS'] || @properties['NR_HOSTS'] || "3").to_i
+@memory = (ENV['VM_MEMORY'] || @properties['VM_MEMORY'] || "6144").to_i
+@cpus = (ENV['VM_CPUS'] || @properties['VM_CPUS'] || "4").to_i
 puts "ml-version=#{@ml_version}"
 puts "nr-hosts=#{@nr_hosts}"
+puts "vm-memory=#{@memory}"
+puts "vm-cpus=#{@cpus}"
 
-def get_vm_name(i)
-  File.basename(Dir.getwd) + "-ml" + i.to_s
+def get_vm_name(i, v)
+  File.basename(Dir.getwd) + "-ml-v" + v + "-" + i.to_s
```

Comment:
ok. Fair enough.

We need to change the Readme for the project then as well :)

On Mon, Feb 23, 2015 at 2:49 PM, Geert <notifications@github.com> wrote:

> In project/Vagrantfile
> <https://github.com/grtjn/mlvagrant/pull/21#discussion_r25163950>:
>
> >
> > -def get_vm_name(i)
> > -  File.basename(Dir.getwd) + "-ml" + i.to_s
> > +def get_vm_name(i, v)
> > +  File.basename(Dir.getwd) + "-ml-v" + v + "-" + i.to_s
>
> You won't be doing that from within one Vagrant project folder. Vagrant
> handles VM name changes badly. And you can always opt to add the ML version
> to the folder name. If you really like to have that version in the name,
> then make it at least an option. Please try to keep existing behavior
> untouched..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grtjn/mlvagrant/pull/21/files#r25163950>.
>